src/components/BoxGrid.tsx

```diff
@@ -3,6 +3,35 @@ import styled from "styled-components"
 
 import Emoji from "./Emoji"
 
+export interface IOpener {
+  isOpen: boolean
+}
+
+export interface IGridStyle {
+  columnNumber: number
+}
+
+export interface IColor {
+  color: string
+}
+
+export interface IBaseItem {
+  emoji: string
+  title: string
+  description: string
+}
+
+export interface IBoxStyle extends IOpener, IGridStyle, IColor {}
```

Comment:
Lets try to keep it simple and avoid creating a lot of interfaces unless we clearly see a purpose for doing that (like reusing those interfaces in other places/components).

For this case, lets just do:
```ts
export interface IBoxStyle {
  isOpen: boolean
  columnNumber: number
  color: string
}
```